contracts/wizards/v1/listing/IListingWizardV1.sol

```diff
@@ -33,4 +53,56 @@ interface IListingWizardV1 is IContractEntity {
         bool immediatePayout,
         uint256 universeId
     ) external returns (uint256 listingId, uint256 listingTermsId);
-}
+
+    /**
+     * @dev Does the same as `createListingWithTerms`, with only difference to allow
+     * delegated listing for the purpose of gas-less TXs.
+     * @param delegatedListingSignature Delegated Listing ECDSA signature ABI encoded (v,r,s)(uint8, bytes32, bytes32).
+     */
+    function delegatedCreateListingWithTerms(
+        Assets.Asset[] calldata assets,
+        Listings.Params calldata params,
+        IListingTermsRegistry.ListingTerms calldata terms,
+        uint32 maxLockPeriod,
+        bool immediatePayout,
+        uint256 universeId,
+        bytes calldata delegatedListingSignature
+    ) external returns (uint256 listingId, uint256 listingTermsId);
+
+    /**
+     * @dev Creates multiple new listings and fill in listing terms on universe level.
+     * Emits an {ListingCreated, UniverseListingTermsRegistered} events for each listing.
+     * @param bulkParams An array of BulkListingParams[] for creating multiple listings.
+     * @return listingIds An array of new listing IDs.
+     * @return listingTermsIds An array of new listing terms IDs.
+     */
+    function createBulkListingWithTerms(
+        BulkListingParams[] calldata bulkParams
+    ) external returns (uint256[] memory listingIds, uint256[] memory listingTermsIds);
+
+    /**
+     * @dev Does the same as `createBulkListingWithTerms`, with only difference to allow
+     * delegated listing for the purpose of gas-less TXs for each asset.
+     * @param bulkParams An array of BulkListingParams[] for creating multiple listings.
+     * @param delegatedListingSignature Delegated Listing ECDSA signature ABI encoded (v,r,s)(uint8, bytes32, bytes32).
```

Comment:
1. delegatedListingSignature rename to delegatedBulkListingSignature  in comments section
2. Fix description for this field 
3.  delegatedListingSignature rename to delegatedBulkListingSignature  in function arguments section